src/backend/tests/risk-edit.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { mockContext } from '../../test-support/test-data/test-utils.stub';
+import { handler } from '../functions/risk-edit';
+
+describe('work package edit', () => {
+  describe('handler', () => {
+    const func = async (event: any) => {
+      return handler(event, mockContext, () => {});
+    };
+
+    describe('validate inputs', () => {
+      const goodBody = {
+        id: 1,
```

Comment:
IDs strings, best to use good example like `ef14de-32bc7`

Reply:
I was not sure how to access that example, but I just used '1' instead of 1. Would that be enough?

Reply:
Yes, but I think copy and pasting my example ID would probably be better to reduce confusion in the future. People might be confused as to why 1 is a string instead of a number. The reality of our database is that the string '1' would not be an actual realistic ID.

Reply:
Even something like "abc" is better just to avoid people thinking it's a number without seeing the quotation marks